maat-court-data-api/src/test/java/gov/uk/courtdata/dces/controller/ConcorContributionsRestControllerTest.java

```diff
@@ -38,23 +40,29 @@ class ConcorContributionsRestControllerTest {
     @Test
     void testContributionFileContent() throws Exception {
 
-        when(concorContributionsService.getConcorFiles(ConcorContributionStatus.ACTIVE))
-                .thenReturn(List.of("FirstXMLFile", "SecondXMLFile", "ThirdXMLFile"));
+        when(concorContributionsService.getConcorContributionFiles(ConcorContributionStatus.ACTIVE))
+                .thenReturn(List.of(
+                        ConcorContributionResponse.builder().concorContributionId(1).xmlContent("FirstXMLFile").build(),
+                        ConcorContributionResponse.builder().concorContributionId(2).xmlContent("SecondXMLFile").build(),
+                        ConcorContributionResponse.builder().concorContributionId(3).xmlContent("ThirdXMLFile").build()));
 
         mvc.perform(MockMvcRequestBuilders.get(String.format(ENDPOINT_URL  +"/concor-contribution-files"))
                         .queryParam("status", ConcorContributionStatus.ACTIVE.name())
                         .contentType(MediaType.APPLICATION_JSON_VALUE))
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$", hasSize(3)))
-                .andExpect(jsonPath("$[0]").value("FirstXMLFile"))
-                .andExpect(jsonPath("$[1]").value("SecondXMLFile"))
-                .andExpect(jsonPath("$[2]").value("ThirdXMLFile"));
+                .andExpect(jsonPath("$.[?(@.concorContributionId==1)].concorContributionId").value(1))
```

Comment:
For the x3 `andExpect` like this:
`.andExpect(jsonPath("$.[?(@.concorContributionId==1)].concorContributionId").value(1))`

It is redundant to check that the `concorContributionId` value is 1. It is sufficient to verify only that it exists
i.e.
`.andExpect(jsonPath("$.[?(@.concorContributionId==1)]").exists())`

This is because the JSON path `"$.[?(@.concorContributionId==1)].concorContributionId"` basically means "Select all elements which are children of the root element ($.) which have a field named concorContributionId which has a value of 1"
So to then assert that the value of `concorContributionId` is 1 is a bit redundant as we already know that the value must be 1 based on the JSON path above.